### PR TITLE
US : some cleaning

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Move chunklists to adaptive
Give BizTV a proper adaptive playlist (compiled all 3 links in a m3u file hosted in iptv-all)
Delete duplicate links

### CODE REVIEW

**Review by MapGuy11 — DISMISSED** [review]

**Review by freearhey — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
@freearhey I was thinking of moving the "VEVO" Amazon-hosted channels into their specific slot. Shall I do that ?

**Comment by freearhey** [comment]
@LaneSh4d0w Which links are we talking about?

**Comment by LaneSh4d0w** [comment]
> @LaneSh4d0w Which links are we talking about?

Links hosted on amazonaws.com

**Comment by freearhey** [comment]
There are many different services running on Amazon AWS, so this criterion would not be enough to separate one from the other.

File: channels/us.m3u [hunk 1]
```diff
@@ -1681,8 +1677,6 @@ https://content.uplynk.com/channel/36953f5b6546464590d2fcd954bc89cf.m3u8
 https://2-fss-1.streamhoster.com/pl_122/201794-1414514-1/playlist.m3u8
 #EXTINF:-1 tvg-id="VictoryTelevisionNetwork.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/yqA2PJj.jpg" group-title="Religious",Victory Television Network (240p)
 http://184.173.179.163:1935/victorytelevisionnetwork/victorytelevisionnetwork/playlist.m3u8
-#EXTINF:-1 tvg-id="VidaMejorTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/W5C1ZRd.png" group-title="",Vida Mejor TV (480p)
-https://uni10rtmp.tulix.tv/bettervida/bettervida.stream/chunklist.m3u8
```
Comment (sguinetti) [inline channels/us.m3u:1685]: VidaMejorTV is 3ABN Latino, right?

Comment (LaneSh4d0w) [reply channels/us.m3u:1685]: We already have 3ABN Latino, which is why I've deleted this stream.

Comment (sguinetti) [reply channels/us.m3u:1685]: No problem.

**Review by sguinetti — APPROVED** [review]

**Review by MapGuy11 — APPROVED** [review]

**Pull request merged**